### DISCUSSION TITLE
Selecting a subset of data based on latitude and longitude distribution

### DISCUSSION THREAD

**Discussion by DaniJonesOcean** [pydata/xarray#5338] [Q&A]
I have an observational dataset that is unevenly distributed in latitude and longitude. Is there a handy xarray method for selecting a subset of such a dataset that is uniformly distributed in latitude and longitude? For example, a good subset would have at least one profile in every 1°x1° grid cell.

**Comment by max-sixty** [comment]
Do you have a copy-pastable example?
